cli_proton_python/sender.py

```diff
@@ -183,6 +183,10 @@ def prepare_message(self):
             msg.id = self.opts.msg_id
         if self.opts.msg_correlation_id is not None:
             msg.correlation_id = self.opts.msg_correlation_id
+        if self.opts.msg_group_id:
+            msg.group_id = self.opts.msg_group_id.decode()
```

Comment:
The use of decode is incompatible with python3 where the strings are unicode by default and str object do not provide decode method, so you may end-up w/ AttributeError. Please use to_unicode callback in options.py instead. You may inspire how to do that ie.: by looking at  'msg-correlation-id' options parsing.